airbyte-cdk/java/airbyte-cdk/db-sources/src/main/kotlin/io/airbyte/cdk/integrations/source/relationaldb/InitialLoadHandler.kt

```diff
@@ -8,12 +8,15 @@ import io.airbyte.commons.util.AutoCloseableIterator
 import io.airbyte.protocol.models.CommonField
 import io.airbyte.protocol.models.v0.AirbyteMessage
 import io.airbyte.protocol.models.v0.ConfiguredAirbyteStream
+import java.time.Duration
 import java.time.Instant
+import java.util.*
```

Comment:
do we need bring in everything from `java.util`?

Reply:
Removed